static/vue.js/workflow/level_1.js

```diff
@@ -136,7 +137,7 @@ new Vue({
 					}
                 );
 				if (response.data) {
-					toastr.success(`${this.level_1_label} successfully saved`);
+					toastr.success(`Program ${this.name} successfully saved`);
```

Comment:
We may not want to hardcode `Program` as we have a way for a user to rename what they call a "Program". I think this is what the `level_1_label` is but I might be wrong. cc: @TAnas0 @Kimaiyo077